metadata.py

```
@@ -383,79 +416,136 @@ def match_publisher(
+    }
+
     dic["description"] = parse_desc(path)
-    dic["file_path"] = f"{path}"
+
     probable_pub = parse_publisher(path)
     if probable_pub:
-        dic["publisher_id"] = match_publisher(probable_pub)
+        try:
+            dic["publisher_id"] = match_publisher(probable_pub)
+        except KeyError as e:
+            logging.warning(f"Publisher not matched: {e}")
```

Ruff (0.11.9)

`warning()` call on root logger

(LOG015)